### ISSUE TITLE
Calendar dates

### ISSUE THREAD

**Issue by d0rable** [Eucharistie/streaming-links#28]
Can we add beside weekMass and sundayMass SPECIAL? For special and important events such as urbi et orbi etc. They appear in another color. Maybe we can also use it for Holy Week or create another one for Holy Week also?

**Comment by duvrai** [comment]
Yes, this is urgent! The schema is changing all the time, take a look at #1
